### DISCUSSION TITLE
用勃勃方法学习英语的一点疑惑

### DISCUSSION THREAD

**Discussion by CourierYoung** [The-Run-Philosophy-Organization/run#328] [Q&A]
我是直接买了寒冰语法还有英英词典，用了两周这个方法以后发现效率实在太低，不是这个方法不行，但是我不会的生词实在太多了，见一个查一个，背了例子和释义后一个词大概要花五分钟的时间（我查阅的时候看到感兴趣的词也会停下来背），我是不是应该改变思路，先把四六级GRE红宝书上面的单词全部背下来再这么干？

**Comment by JanuaryDesk** [comment]
有高中3500词基础也不会“不会的生词太多了吧”。。

**Reply by CourierYoung** [reply]
我玩极乐迪斯科根本玩不动啊，光技能就花了我一小时背完

**Reply by sgfcz** [reply]
极乐迪斯科保守单词量要在2w5到3w你是哪来的信心。。。。。

**Reply by sgfcz** [reply]
美国人自己都玩不明白这个游戏
